5-WebApp-AuthZ-Roles-Groups/Controllers/HomeController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System.Diagnostics;
+using System.Security.Claims;
+using WebApp_OpenIDConnect_DotNet.Models;
+
+namespace WebApp_OpenIDConnect_DotNet.Controllers
+{
+    [Authorize]
+    public class HomeController : Controller
```

Comment:
I am assuming you are referring to the UserProfileController. Since it has code to show how to handle overages, i kept it away from index. 

---
In reply to: [272943732](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/84#discussion_r272943732) [](ancestors = 272943732)